cluster/saltbase/salt/kube-master-addons/kube-master-addons.sh

```diff
@@ -15,28 +15,47 @@
 # limitations under the License.
 
 # loadedImageFlags is a bit-flag to track which docker images loaded successfully.
-let loadedImageFlags=0;
+let loadedImageFlags=0
+restart_docker=false
 
 while true; do
- if which docker 1>/dev/null 2>&1; then
-   if docker load -i /srv/salt/kube-bins/kube-apiserver.tar 1>/dev/null 2>&1; then
-     let loadedImageFlags="$loadedImageFlags|1";
-   fi;
-   if docker load -i /srv/salt/kube-bins/kube-scheduler.tar 1>/dev/null 2>&1; then
-     let loadedImageFlags="$loadedImageFlags|2";
-   fi;
-   if docker load -i /srv/salt/kube-bins/kube-controller-manager.tar 1>/dev/null 2>&1; then
-     let loadedImageFlags="$loadedImageFlags|4";
-   fi;
- fi;
+  if which docker 1>/dev/null 2>&1; then
+
+     /etc/kubernetes/alarm 200 docker load -i /srv/salt/kube-bins/kube-apiserver.tar 1>/dev/null 2>&1
+     rc=$?
+     if [[ $rc == 0 ]]; then
+       let loadedImageFlags="$loadedImageFlags|1"
+     elif [[ $rc == 124 ]]; then
```

Comment:
Document where 124 comes from, and preferrably just:
```
declare -r FOO_MAGIC_RETURN=124
```
somewhere as readable documentation.